base/util.go

```diff
@@ -139,6 +139,10 @@ func ToInt64(value interface{}) (int64, bool) {
 	return 0, false
 }
 
+func IsPseudoDoc(ID string) bool {
+	return strings.HasPrefix(ID, "_user/") || strings.HasPrefix(ID, "_role/")
```

Comment:
I'm pretty sure that appendUserFeed (and appendVectorUserFeed) are the only places that create these pseudo-user docs).  The LogEntries that are created using the same naming convention in processPrincipalDoc don't make it to any channels, AFAIK, so shouldn't be making it to the changes feed.

It makes sense to include a unit test to validate this, though.